tsconfig.json

```diff
@@ -24,8 +24,9 @@
     },
     "resolveJsonModule": true,
     "skipLibCheck": true,
-    "rootDir": "src"
+    "rootDir": "src",
     //"strict": true,
+    "downlevelIteration": true
```

Comment:
Curios as why this is needed.

Reply:
@imbhargav5  It is needed to rest spread a map like `[...map]`.